src/common.speech/DialogServiceAdapter.ts

```diff
@@ -792,6 +792,7 @@ export class DialogServiceAdapter extends ServiceRecognizerBase {
             context: {
                 interactionId: guid
             },
+            messagePayload: this.activityTemplate ? this.activityTemplate : undefined,
```

Comment:
This sounds like your property is a string hence it will be a Json string and not a Json object. This is what I expect but I want to make sure this is what you wanted.